### PR TITLE
Add loading option to controller

### PR DESCRIPTION
**Pull request by bierik**
⚠️  Depends on https://github.com/4teamwork/plonetheme.teamraum/pull/462

Closes https://github.com/4teamwork/opengever.core/issues/1990

This new behaviour of the controller makes it possible to turn a button in a loading button.

### CODE REVIEW

File: opengever/meeting/browser/resources/meeting.js [hunk 1]
```diff
@@ -352,7 +348,8 @@
         target: ".schedule-proposal",
         callback: this.schedule,
         options: {
-          update: true
+          update: true,
+          loading: true
         }
       },
```
Comment (deiferni) [inline opengever/meeting/browser/resources/meeting.js]: what about `schedule-paragraph`?

File: opengever/meeting/browser/resources/meeting.js [hunk 2]
```diff
@@ -352,7 +348,8 @@
         target: ".schedule-proposal",
```
Comment (deiferni) [inline opengever/meeting/browser/resources/meeting.js]: i just noticed that when scheduling a proposal/section with an empy text-field a request is sent which results in a server-side error. did the behavior change in this PR or is this an unrelated issue?

File: opengever/meeting/browser/resources/meeting.js (hunk 1 above)
Comment (bierik) [inline opengever/meeting/browser/resources/meeting.js]: When the user adds a paragraph the input field is cleaned. So the user is not able to add the same paragraph again anyway.

**Comment by bierik** [comment]
@deiferni updated

**Pull request merged**